templates/leaderboard/includes/list.html

```diff
@@ -1,18 +1,61 @@
+{% load staticfiles %}
 {% load humanize %}
 {% load perdiem %}
 
-{% if leaders %}
     <div class="leaderboard">
-        <h4>Top {{ name }}</h4>
-        {% for leader_context in leaders %}
-            <a href="{{ leader_context.url }}">
-                <div class="leaderboard-padding button">
-                    <img class="profile-crop" src="{{ leader_context.avatar_url }}" alt="{{ leader_context.name }}" />
-                    <br/>
-                    <h4 class="chart-name">#{{ forloop.counter }}: {{ leader_context.name }}</h4>
-                    <h4 class="chart-amount">${{ leader_context.amount|notrail_floatformat:2|intcomma }}</h4>
-                </div>
-            </a>
-        {% endfor %}
+        <h2>Investors</h2>
+        <h4>Top earnings and investments</h4>
+        <div class="break">
+            <hr>
+        </div>
+        <div class="chart-container">
+            <canvas id="myChart"></canvas>
+        </div>
+        <h2>Artists</h2>
+        <h4>Top earnings and investments</h4>
+        <div class="break">
+            <hr>
+        </div>
+        <div class="chart-container">
+            <canvas id="myChart2"></canvas>
+        </div>
     </div>
-{% endif %}
+
+{% block extrajs %}
+    <script type="text/javascript" src="{% static "js/vendor/Chart.min.js" %}"></script>
+
+    <script>
+            var ctx = document.getElementById("myChart").getContext('2d');
+            var myChart = new Chart(ctx, {
+              type: 'bar',
+              data: {
+                labels: [{% if top_earned_investors %}{% for top_earned_investors_context in top_earned_investors %}"{{ top_earned_investors_context.name }}"{% if not forloop.last %},{% endif %}{% endfor %}{% endif %}],
+                datasets: [{
+                  label: 'Amount Invested',
+                  data: [{% if top_invested %}{% for top_invested_context in top_raised %}{{ top_invested_context.amount|notrail_floatformat:2 }}{% if not forloop.last %},{% endif %}{% endfor %}{% endif %}],
+                  backgroundColor: "#85929E"
+              }, {
+                label: 'Amount Earned',
+                data: [{% if top_earned_investors %}{% for top_earned_investors_context in top_earned_investors %}{{ top_earned_investors_context.amount|notrail_floatformat:2 }}{% if not forloop.last %},{% endif %}{% endfor %}{% endif %}],
+                backgroundColor: "#4cbd7e"
+              }]
+              }
+            });
+            var ctx = document.getElementById("myChart2").getContext('2d');
```

Comment:
Updated the chart names. I thought the ctx had to be the same name for the charts to work properly.